webforces/server/structs.py

```diff
@@ -1,5 +1,12 @@
 from dataclasses import dataclass, field
 from typing import List
+from enum import Enum
+
+
+class DBStatuses(Enum):
```

Comment:
Isn't `DBStatus` name better in this case? What do you think?

Reply:
Changed to` DBStatus`.